pipelines/skylab/multiome/atac.wdl

```diff
@@ -24,13 +24,19 @@ workflow ATAC {
     
     # BWA ref
     File tar_bwa_reference
+    # BWA machine type -- to select number of splits 
```

Comment:
Are all these values truly independent? That is can I change only 1 at a time or do some depend on others? We want to scope it to the minimum inputs that allow the user the flexibility needed

Reply:
Usually the user will probably only need to change the number of threads, memory and cpu platform. That is usually enough to specify the machine. I thought it would be a good idea to add the rest (with default values) to allow user flexibility just incase it needs to be changed. I could change it to only include those three as input and add others as optional parameters

Reply:
Changed to take only the three parameters to provision same VM as BWA as we discussed with Intel